Directory.Packages.props

```diff
@@ -45,7 +45,11 @@
     <PackageVersion Include="System.Reflection.Metadata" Version="8.0.0" />
     <PackageVersion Include="System.Resources.Extensions" Version="8.0.0" />
     <PackageVersion Include="System.Runtime.CompilerServices.Unsafe" Version="6.0.0" />
-    <PackageVersion Include="TomsToolbox.Wpf.Styles" Version="2.17.3" />
+    <PackageVersion Include="TomsToolbox.Wpf.Composition" Version="2.18.1" />
+    <PackageVersion Include="TomsToolbox.Wpf.Styles" Version="2.18.1" />
     <PackageVersion Include="coverlet.collector" Version="6.0.2" />
   </ItemGroup>
+  <ItemGroup>
+    <GlobalPackageReference Include="TomsToolbox.Composition.Analyzer" Version="2.18.1" />
```

Comment:
Is that really a good idea? That will then be applied to all the VSIX projects, ilspycmd, the installer, et cetera. We have fewer MEF projects than non-MEF projects.

Reply:
GlobalPackageReferences are compile-time only by default, and this is just an analyzer helping to make consistent export definitions.